addons/ballistics/stringtable.xml

```diff
@@ -2279,5 +2279,24 @@
             <Chinese>[ACE] 彈藥補給箱</Chinese>
             <Chinesesimp>[ACE] 弹药补给箱</Chinesesimp>
         </Key>
+        <Key ID="STR_ACE_Ballistics_statBarrelTwist">
+            <English>Barrel twist</English>
+        </Key>
+        <Key ID="STR_ACE_Ballistics_statBarrelLength">
+            <English>Barrel length</English>
+            <French>Longueur du canon</French>
+        </Key>
+        <Key ID="STR_ACE_Ballistics_statAmmo">
```

Comment:
`str_dn_ammo` ?

Reply:
?

Reply:
`str_dn_ammo` `<English>Ammo</English>` - does it suit here?

Reply:
I guess the already existing vanilla translation.